config/prod/arvados-wf-testing.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v3.j2
```

Comment:
there is a new example template for linux instances.  please take a look at https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/3/files

Reply:
Sorry about that, should be fixed!